src/ray/stats/metric_defs.h

```diff
@@ -79,20 +75,26 @@ static Histogram ProcessStartupTimeMs("process_startup_time_ms",
                                       "Time to start up a worker process.", "ms",
                                       {1, 10, 100, 1000, 10000});
 
-static Gauge AvgNumScheduledTasks(
-    "avg_num_scheduled_tasks",
-    "Number of tasks that are queued on this node per second. It doesn't guarantee "
-    "that the task will be executed in this node. If the task is executed, it is "
-    "recorded as avg_num_executed_tasks. If the task is not executed and needs to be "
-    "scheduled in other nodes, it will be recorded as avg_num_spilled_back_tasks",
-    "tasks");
+static Count NumWorkersStarted(
+    "_internal_num_processes_started",
+    "The total number of worker processes the worker pool has created.", "processes");
+
+static Gauge NumInfeasibleTasks(
+    "num_infeasible_tasks",
+    "The number of tasks in the scheduler that are in the 'infeasible' state.", "tasks");
+
+static Count NumQueuedTasks(
```

Comment:
I think that this cumulative count should be called `NumReceivedTasks`, and that we should have a `NumQueuedTasks` gauge that records the queue length (number of tasks in `tasks_to_schedule_`).
```suggestion
static Count NumReceivedTasks(
```